pages/FAQ/_index.md

```diff
@@ -10,121 +11,109 @@ Possible causes:
 GitHub pages).
 
-For a more complete utility, try our own
+For a more complete utility, try our own screenshotting utility:
 [Grimblast](https://github.com/hyprwm/contrib).
 
-### Screenshare / OBS no worky!
+For recording videos, wf-recorder or OBS Studio could be used.
+
+# Screenshare / OBS no worky
 
 Check [Screensharing](../Useful-Utilities/Screen-Sharing).
 
-### How do I change my wallpaper?
+# How do I change my wallpaper?
 
 See [Wallpapers](../Useful-Utilities/Wallpapers).
 
-### My workspace (2, 3, or any other) is like... bugged?
-
-You did the below, unknowingly.
-
-### My games work poorly, especially proton ones!
+# My games work poorly, especially proton ones
 
 Use `gamescope`, tends to fix any and all issues with wayland/Hyprland.
 
-### How heavy is this?
+# How heavy is this?
 
 Not that much heavier than Xorg. If you want maximum performance, consider
 turning off the blur and animations.
 
-### Blur makes my GPU cry :(
-
-You probably forgot to turn on `decoration:blur_new_optimizations`. Thank me
-later.
-
-### My monitor no worky!
+# My monitor no worky
 
 Try changing the mode in your config. If your preferred one doesn't work, try a
 lower one. A good way to list all modes is to get `wlr-randr` and do a
 `wlr-randr --dryrun`
 
-### How do I get a coredump?
+# How do I update?
 
-*These instructions are ONLY for systemd. If you use anything else, you should
-know what you're doing.*
-
-Launch `coredumpctl` in a terminal. Press <key>END</key> on the keyboard to go
-to the end. Note the **last** (the one furthest to the bottom) crash that has
-`/usr/bin/Hyprland` as an executable. Remember the PID of it (the first number
-after the date in a given line) exit (<key>Ctrl</key>+<key>C</key>) type
-`coredumpctl info PID` where `PID` is the remembered PID. Send the entire thing
-as a file.
-
-### How do I update?
-
-open a terminal where you cloned the repo.
-`git pull && sudo make clear && sudo make install`
-
-### Waybar popups render behind the windows??
+Open a terminal where you cloned the repo.
+```bash
+git pull 
+sudo make clear 
+sudo make cleaninstall
+```
 
-You have misconfigured Waybar. Make sure the `layer` in the waybar config is set to `top`, and not `bottom`.
+If you are using the AUR (hyprland-git) package, you
+will need to cleanbuild to update the package. Paru
+has been problematic with updating before, use Yay.
 
-### How do I screen lock?
+# How do I screen lock?
 
 Use a wayland-compatible locking utility using WLR protocols, e.g. `swaylock`.
 
-### How do I change me mouse cursor?
+# How do I change me mouse cursor?
 
 Use a tool like for example `lxappearance` to change the GTK cursor.
 
 After that, add `exec-once=hyprctl setcursor [THEME] [SIZE]` to your config and
 restart Hyprland.
 
+About QT applications, instead, Hyprland automatically exports `XCURSOR_SIZE`, 24 
+being the default. You may export a different value in your wrapper to change it.
```

Comment:
*For QT applications, Hyprland exports `XCURSOR_SIZE` as 24, which is the default. You can overwrite this by exporting `XCURSOR_SIZE` to a different value in your wrapper.*